src/components/UI/Header/CTA/CTA.jsx

```diff
@@ -2,29 +2,43 @@ import "./CTA.css";
 import { Link } from "react-router-dom";
 import { useLocation } from "react-router-dom";
 import { ThemeToggle } from "../../ThemeToggle/ThemeToggle";
+import { useAuth } from "../../../../contexts";
 
 export const CTA = () => {
+	const { isAuth, toggleAuth } = useAuth();
+
 	const { pathname } = useLocation();
 
+	const onClickHandler = () => {
+		if (isAuth) {
+			localStorage.removeItem("noteToken");
+			toggleAuth();
+		}
+	};
+
```

Comment:
User proper naming so onClickHandler could have used different names related to the context.

Reply:
Will improve naming.